specs/airr-schema.yaml

```diff
@@ -2267,9 +2266,16 @@ Clone:
             description: Identifier to the associated repertoire in study metadata.
             x-airr:
                 miairr: false
+                required: true
         data_processing_id:
             type: string
             description: Identifier for the data processing object.
+            x-airr:
+                miairr: false
+                required: true
+        sequence_id:
+            type: list
```

Comment:
@scharch can you change this to be [array as in this link](https://cswr.github.io/JsonSchema/spec/arrays/) and then I can merge?

Reply:
@eharkins done :)

Reply:
@scharch thanks! I'll merge now. Should we add the type of items in the array like they do in the docs I linked:
```
 "items": {
           "type": "string"
          }
```
(or other restrictions on what the array can contain)?

Reply:
> Should we add the type of items in the array like they do in the docs I linked:

dunno, sounds like a question for @schristley on what our typical format/standard for such things is...

Reply:
The jsonschema array docs suggest you might be required to specify the `items` definition, and regardless it seems like a good idea if we know the expected type of things in the array as we probably do with `sequence_id` here. Also most of the array fields in the existing AIRR schemas have an `items` definition [e.g.](https://github.com/airr-community/airr-standards/blob/master/specs/airr-schema.yaml#L1290)

I can add one directly to the merged `clones-and-trees-release` branch.